test-packages/type-tests/package.json

```
@@ -8,9 +8,8 @@
   "types": "test",
   "repository": "github:SAP/cloud-sdk-js",
   "scripts": {
-    "test": "yarn test:self-test && yarn test:type",
+    "test": "yarn test:type",
     "test:type": "dtslint --expectOnly --localTs ../../node_modules/typescript/lib test",
```

```suggestion
    "test": "dtslint --expectOnly --localTs ../../node_modules/typescript/lib test",
```
[pp] I think we can just always use the `test` without `test:type`.